### PR TITLE
Typos and code style improvements

### PR DESCRIPTION
**Pull request by jakubjezek001**
There is a lot of changes but most of them are about improving code style and typos.

Also as you can see I am pushing the ShotGrid namespace rather then Shotgrid. It looks nicer but this is really just my opinion ;)

### CODE REVIEW

**Comment by fabiaserra** [comment]
please don't try merge this PR until the bigger PRs https://github.com/ynput/ayon-shotgrid/pull/72, https://github.com/ynput/ayon-shotgrid/pull/79 are merged. And like @m-u-r-p-h-y said, if anything we should update to Flow now

File: services/shotgrid_common/utils.py [hunk 1]
```diff
@@ -64,16 +64,17 @@ def _sg_to_ay_dict(sg_entity: dict, project_code_field=None) -> dict:
         folder_type = sg_entity["type"]
 
     ay_dict = {
-        "type": ay_entity_type, # In the EH there are either `folder_type` or `task_type`.
+        # In the EntityHub there are either `folder_type` or `task_type`
```
Comment (fabiaserra) [inline services/shotgrid_common/utils.py:67]: this is not readable anymore if it's not a line comment. However, I think we shouldn't limit this sync to just these two types

File: services/shotgrid_common/ayon_shotgrid_hub/update_from_shotgrid.py [hunk 2]
```diff
@@ -71,18 +77,11 @@ def create_ay_entity_from_sg_event(sg_event, sg_project, sg_session, ayon_entity
...
     )
-    logging.debug(f"SG Entity as Ayon dict: {sg_entity_dict}")
-    if not sg_entity_dict:
-        logging.warning(
-            "Entity {sg_event['entity_type']} <{sg_event['entity_id']}> "
-            "no longer exists in Shotgrid, aborting..."
-        )
-        return
-
+    logging.debug(f"ShotGrid Entity as Ayon dict: {sg_entity_dict}")
```
Comment (iLLiCiTiT) [inline services/shotgrid_common/ayon_shotgrid_hub/update_from_shotgrid.py:80]:
```suggestion
    logging.debug(f"ShotGrid Entity as AYON dict: {sg_entity_dict}")
```

File: services/shotgrid_common/utils.py [hunk 3]
```diff
@@ -18,13 +18,13 @@
 
 
 def _sg_to_ay_dict(sg_entity: dict, project_code_field=None) -> dict:
-    """Morph a Shotgrid entity dict into an Ayon's Entity Hub compatible one.
+    """Morph a ShotGrid entity dict into an Ayon's Entity Hub compatible one.
```
Comment (iLLiCiTiT) [inline services/shotgrid_common/utils.py:21]:
```suggestion
    """Morph a ShotGrid entity dict into an ayon-api Entity Hub compatible one.
```

File: server/__init__.py [hunk 4]
```diff
@@ -52,7 +55,7 @@ async def create_shotgrid_attributes(self) -> bool:
 
         shotgrid_attributes = await Postgres.fetch(
             "SELECT name from public.attributes "
-            f"WHERE (name = '{SG_ID_ATTRIB}' OR name = '{SG_TYPE_ATTRIB}' OR name = '{SG_PUSH_ATTRIB}') "
+            f"WHERE (name = '{SG_ID_ATTRIB}' OR name = '{SG_TYPE_ATTRIB}' OR name = '{SG_PUSH_ATTRIB}') "  # noqa
```
Comment (iLLiCiTiT) [inline server/__init__.py:58]:
```suggestion
            f"WHERE (name = '{SG_ID_ATTRIB}'"
            f" OR name = '{SG_TYPE_ATTRIB}'"
            f" OR name = '{SG_PUSH_ATTRIB}') "
```

**Comment by jakubjezek001** [comment]
We are not planning to rename it any time soon to Flow

**Comment by fabiaserra** [comment]
I don't get why did you merge this? Didn't I ask yesterday to please hold on this until the bigger PRs are merged? now it makes this pending PR https://github.com/ynput/ayon-shotgrid/pull/79 harder to merge and https://github.com/ynput/ayon-shotgrid/pull/72 contains a lot of things not specific to those changes

**Pull request merged**